linera-base/src/data_types.rs

```diff
@@ -1125,16 +1128,11 @@ impl Serialize for Blob {
     {
         if serializer.is_human_readable() {
             let blob_bytes = bcs::to_bytes(&self.content).map_err(serde::ser::Error::custom)?;
-            serializer.serialize_str(&format!(
-                "{}:{}",
-                self.id.blob_type,
-                hex::encode(blob_bytes)
-            ))
+            serializer.serialize_str(&hex::encode(blob_bytes))
         } else {
             SerializableBlob::serialize(
                 &SerializableBlob {
-                    blob_type: self.id.blob_type,
-                    blob_content: self.content.clone(),
+                    content: self.content.clone(),
```

Comment:
This clone is a bit unfortunate. Actually, it looks like you can just use `BlobContent::serialize` directly.